packages/expo-doctor/src/doctor.ts

```diff
@@ -130,28 +122,11 @@ export async function runChecksAsync(
 }
 
 export function getChecksInScopeForProject(exp: ExpoConfig, pkg: PackageJSONConfig) {
-  // add additional checks here
-  const checks = [
-    new PackageManagerVersionCheck(),
-    new IllegalPackageCheck(),
-    new GlobalPackageInstalledLocallyCheck(),
-    new SupportPackageVersionCheck(),
-    new ExpoConfigSchemaCheck(),
-    new ExpoConfigCommonIssueCheck(),
-    new DirectPackageInstallCheck(),
-    new PackageJsonCheck(),
-    new ProjectSetupCheck(),
-    new MetroConfigCheck(),
-    new NativeToolingVersionCheck(),
-    new AppConfigFieldsNotSyncedToNativeProjectsCheck(),
-    new StoreCompatibilityCheck(),
-  ];
-
   if (getReactNativeDirectoryCheckEnabled(exp, pkg)) {
     chalk.yellow(
       'Enabled experimental React Native Directory checks. Unset the EXPO_DOCTOR_ENABLE_DIRECTORY_CHECK environment variable to disable this check.'
     );
-    checks.push(new ReactNativeDirectoryCheck());
+    DOCTOR_CHECKS.push(new ReactNativeDirectoryCheck());
```

Comment:
Feels off that we're modifying an import here, that's also named like a constant. Extracting the individual checks out of this file is probably good, but probably a different approach would be better. Maybe there's a separate `inScopeCheckResolver` that incorporates env variables, doctor config, SDK version, and outputs the final checks. It could be mostly a straight encapsulation of what's already here, and would at least make doctor.ts cleaner.